llvm/lib/Target/X86/MCTargetDesc/X86MCCodeEmitter.cpp

```diff
@@ -217,6 +271,12 @@ class X86OpcodePrefixHelper {
     case REX:
       emitByte(0x40 | W << 3 | R << 2 | X << 1 | B, CB);
       return;
+    case REX2:
+      emitByte(0xD5, CB);
+      emitByte(M << 7 | R2 << 6 | X2 << 5 | B2 << 4 | W << 3 | R << 2 | X << 1 |
+                   B,
+               CB);
```

Comment:
indentation

Reply:
Indentation here is correct here. clang-format makes code look like this.

Reply:
Done